src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+@font-face {
+  font-family: avenir;
+  src: url(../src/fonts/Avenir.ttc), url(../src/fonts/Avenir-Heavy.ttf), url(../src/fonts/Avenir-Book.ttf);
+}
+
+.heading {
+  display: none;
+}
+
+.search-bar {
+  width: 100%;
+  font-family: avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.form {
+  position: relative;
+}
+
+.magnify1 {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+}
+
+.magnify2 {
+  position: absolute;
+  left: 13px;
+  top: 35.71%;
+  height: 11px;
+}
+
+.space {
+  height: 12px;
+}
+
+.space1 {
+  height: 20px;
+}
+
+.search-bar::placeholder {
+  font-family: avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7;
+  border-radius: 4px;
```

Comment:
these rules are redundant – you've already put the same rules under `.search-bar` selector